index.html

```diff
@@ -40,7 +38,7 @@ <h2 id="main-h2">Private Digital Currency</h2>
                     <div class="row middle-xs info-block-row private">
                         <div class="col-lg-6 col-md-6 col-sm-8 col-xs-12  why-text">
                             <h3>Monero is secure.</h3>
-                            <p>Monero is a decentralized cryptocurrency, meaning it is extremely difficult to hack and steal your funds. Transactions are confirmed by distributed consensus, and then immutably recorded on the blockchain. Third-parties do not need to be trusted to keep your Monero safe. </p>
+                                <p>Monero is a decentralized cryptocurrency, meaning secure digital cash powered by a network operated by its users. Transactions are confirmed by distributed consensus, and then immutably recorded on the blockchain. You are the bank - no one knows how much you have or how you spend it. </p>
```

Comment:
Monero is a decentralized cryptocurrency, meaning, like Bitcoin, it is extremely difficult to hack and steal your funds. Transactions are confirmed by distributed consensus, and then immutably recorded on the blockchain. Your are your own bank and third parties do not need to be trusted to keep your Monero safe.